code/game/machinery/bots/secbot.dm

```diff
@@ -384,6 +384,22 @@ Auto Patrol: []"},
 		target = user
 		mode = BOT_HUNT
 
+/obj/machinery/bot/secbot/Crossed(atom/movable/AM)
+	if(ismob(AM) && target)
+		var/mob/living/carbon/C = AM
+		if(!istype(C) || !C || in_range(src, target))

```

Comment:
I don't see why the  in_range(src, target) is necessary, beepsky doesn't move when he cuffs someone, so he cannot cross someone.